packages/gatsby-source-nacelle/package.json

```diff
@@ -21,14 +21,14 @@
   "author": "Nacelle Inc. (getnacelle.com)",
   "license": "ISC",
   "dependencies": {
-    "@nacelle/client-js-sdk": "^3.1.0",
     "@nacelle/contentful-preview-connector": "^0.1.5",
+    "@nacelle/storefront-sdk": "^1.0.1",
     "lodash.get": "^4.4.2",
     "lodash.set": "^4.3.2"
   },
   "peerDependencies": {
     "gatsby": ">2.0.31",
```

Comment:
Should we require that everyone use `gatsby > 4.0.9` at this point? Just to make sure we don't have any issues with "old" Gatsby versions, and we're making a breaking change anyways.

cc @NWRichmond for input

Reply:
I think `">4" is probably a safe move. I know that our Gatsby customers unanimously want to use Gatsby 4 at this point, so removing implied support for lesser versions doesn't feel risky.